ts/utils/typing.ts

```diff
@@ -0,0 +1,20 @@
+import * as t from "io-ts";
```

Comment:
If I'm correct, this function allows to check that a certain keys is part of an object. Why we should use this instead of a simpler `object[key] !== undefined`?
And in case we want to type-check at runtime we could do as follows: 
```
const isOfMyType = (u: unknown): u is MyType => object[u as MyType] !== undefined;
```